website/source/docs/provisioners/puppet-masterless.html.markdown

```diff
@@ -64,6 +65,11 @@ Optional parameters:
   the `manifest_file`. It is a separate directory that will be set as
   the "manifestdir" setting on Puppet.
 
+  ~> `manifest_dir` is passed to `puppet apply` as the `--manifestdir` option.
+     This option was deprecated in puppet 3.6, and is slated to be removed in
+     puppet 4.0. If you have multiple manifests you should simply use
```

Comment:
Is removed in Puppet 4.X: https://docs.puppetlabs.com/references/4.stable/configuration.html#manifest